marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/hub-entities.xqy

```diff
@@ -235,6 +237,28 @@ declare %private function hent:fix-options-exp($nodes as node()*, $sortable-prop
       default return $n
 };
 
+declare %private function hent:add-sort-operators-to-search-options($sortable-properties as map:map)
+{
+  let $sort-operators :=
+    for $property-path in map:keys($sortable-properties)
+      let $property-path := fn:replace($property-path, "es:", "*:")
```

Comment:
Are there tests for this line of code? This seems brittle - what if the the namespace prefix is "aes"? Then an occurrence of "aes:" would end up as "a*:", which I believe will fail. 

I recommend putting the conversion of a property path into a constraint name into a separate non-private function and writing unit tests for that. That will both verify and document the behavior of the function.

Reply:
Also, see my comment below - I don't think we need property paths here, because for 5.3.0, we're only supporting sorting on top-level properties. Thus, a simple property name will suffice. And I don't think that should have a namespace prefix in it, in which case there's no need to replace "es:".